jina/peapods/networking.py

```diff
@@ -35,13 +36,18 @@ def add_connection(self, address: str):
         :param address: Target address of this connection
         """
         if address not in self._address_to_connection_idx:
+            parsed_address = urlparse(address)
```

Comment:
Sometimes addresses are names when docker compose or kubernetes is used. Are we sure this will not raise an exception there?

Reply:
addressed